wrapper/build.gradle.kts

```diff
@@ -206,86 +207,142 @@ tasks.jar {
     }
 }
 
+junitHtmlReport {
+    // The maximum depth to traverse from the results dir.
+    // Any eligible reports will be included
+    maxDepth = 9
+
+    //RAG status css overrides
+    cssRed = "red"
+    cssAmber = "orange"
+    cssGreen = "green"
+
+    //Processing directories
+    testResultsDir = "test-results"
+    testReportsDir = "report"
+
+    //Fail build when no XML files to process
+    isFailOnEmpty = true
+}
+
 tasks.withType<Test> {
+    dependsOn("jar")
+    testLogging {
+        this.showStandardStreams = true
+    }
     useJUnitPlatform()
 
+    System.getProperties().forEach {
+        if (it.key.toString().startsWith("test-no-")) {
+            systemProperty(it.key.toString(), it.value.toString());
+        }
+    }
+
+    // Disable the test report for the individual test task
+    reports.junitXml.required.set(true)
+    reports.html.required.set(false)
+
     systemProperty("java.util.logging.config.file", "${project.buildDir}/resources/test/logging-test.properties")
-}
 
-// Run Aurora Postgres integrations tests in container
-tasks.register<Test>("test-integration-aurora-postgres") {
-    group = "verification"
-    filter.includeTestsMatching("integration.host.AuroraPostgresContainerTest.runTestInContainer")
+    finalizedBy("junitHtmlReport")
 }
 
-tasks.register<Test>("test-performance-aurora-postgres") {
+tasks.register<Test>("test-all-environments") {
     group = "verification"
-    filter.includeTestsMatching("integration.host.AuroraPostgresContainerTest.runPerformanceTestInContainer")
+    filter.includeTestsMatching("integration.refactored.host.TestRunner.runTests")
+    doFirst {
+        systemProperty("test-no-performance", "true")
+    }
 }
 
-// Run standard Postgres tests in container
-tasks.register<Test>("test-integration-standard-postgres") {
+tasks.register<Test>("test-all-docker") {
```

Comment:
nit: to avoid verbosity with task names, should we change to just `test-docker`, `test-performance`, `test-aurora`, `debug-docker`, `debug-performance`, `debug-aurora`?